src/styles/stars.css

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
```

Comment:
you should have all the modifiers, from 1 to 5

Reply:
I really don't understand how it works! Why we need add all modifiers?

Reply:
in the morning I understood everything